boards/arm64/mimx93_evk/mimx93_evk_a55_sof_defconfig

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# ARM Options
+CONFIG_AARCH64_IMAGE_HEADER=y
+CONFIG_ARMV8_A_NS=y
+
+# The number of bits used for PAs and VAs
+# from Zephyr needs to match the number of
+# bits used for said address used by Jailhouse.
+CONFIG_ARM64_VA_BITS_40=y
+CONFIG_ARM64_PA_BITS_40=y
+
+# Cache Options
+CONFIG_CACHE_MANAGEMENT=y
+CONFIG_ICACHE_LINE_SIZE_DETECT=y
+# Currently, SOF doesn't allow for run-time
+# data cache line size detection.
+CONFIG_DCACHE_LINE_SIZE_DETECT=n
```

Comment:
same with `CONFIG_XIP`, the default of `CONFIG_DCACHE_LINE_SIZE_DETECT` is `n`. But since this CONFIG might be a kind of alert, I am not sticking to it.

Reply:
Fixed. Added a more descriptive message which should itself serve as a warning.